### DISCUSSION TITLE
Navigate between dictionaries in suggestions?

### DISCUSSION THREAD

**Discussion by plattfot** [minad/jinx#220] [Q&A] [answered]
Hi,

First of thank you for writing jinx. As a bilingual user, it was a pain to use ispell to toggle between dictionaries when writing.  Especially since I tend to mix Swedish and English in my notes. Having it use both dictionaries at the same time is such a great feature — I even threw in `en_CA` to mix things up.

One thing that I haven't figured out is how to quickly navigate between the dictionaries when correcting a word. I have `jinx-languages` set to `en_US sv_SE en_CA`.  For example, if I want to correct a word in Swedish. I need to scroll through all the `en_US` suggestions before getting to the Swedish ones.  And if I want to check the Canadian spelling, I need to scroll through both `en_US` and `sv_SE`.

While I could call `jinx-languages` and change the order, it is a bit cumbersome to do that if I just want to correct a word or two. Would be better if I could quickly jump to the different `Suggestions from dictionary…`. But I haven't figured out how to do that.

Is there a way to do that? Or am I missing something obvious?

Thanks

**Comment by minad** [comment, marked as answer]
It depends on the completion UI which you use. In Vertico you can use the `vertico-next-group` command. Note that you can also select entries directly by pressing the corresponding digit.

**Reply by plattfot** [reply]
I'm using Vertico and `vertico-next-group` command was exactly what I was looking for. Thanks!
